config.yaml.defaults

```diff
@@ -60,6 +60,20 @@ ports:
     websocket_path_in: 'ipc://run/message_flow_in'
     websocket_path_out: 'ipc://run/message_flow_out'
 
+
+external_logging:
+    enabled: False
+    services:
+        papertrail:
```

Comment:
Do we need both global enable and service-specific enable?

Reply:
it's a convenience move if we have many services. For now it's superfluous.